.github/workflows/linux-eic-shell.yml

```
@@ -104,13 +104,13 @@ jobs:
         name: build-clang-eic-shell-Debug
         path: build/
     - uses: eic/run-cvmfs-osg-eic-shell@main
-      if: ${{ github.event_name == 'pull_request'}}
+      if: ${{ github.event_name == 'pull_request' && ! contains(github.event.comment.body, 'run_clang_tidy') }}
```

Nevermind, this requires `on: issue_comment` which we don't want.
```suggestion
      if: ${{ github.event_name == 'pull_request' }}
```